src/screens/Apps/Lendasat/Index.tsx

```diff
@@ -0,0 +1,202 @@
+              hasNpub: false,
+            },
+            ark: {
+              canSend: true,
+              canReceive: true,
+            },
+          }
+        },
+        async onSendToAddress(address: string, amount: number, asset: 'bitcoin' | LoanAsset): Promise<string> {
+          if (!svcWallet) {
+            throw Error('Wallet not initialized')
+          }
+
+          switch (asset) {
+            case 'bitcoin':
+              if (isArkAddress(address)) {
+                const txId = await svcWallet?.sendBitcoin({ amount: amount, address: address })
+                if (txId) {
+                  return txId
+                } else {
+                  throw new Error('Unable to send bitcoin')
+                }
+              } else if (isBTCAddress(address)) {
+                return await collaborativeExit(svcWallet, amount, address)
+              } else {
+                throw Error(`Unsupported address ${address}`)
+              }
+            case 'UsdcPol':
+            case 'UsdtPol':
+            case 'UsdcEth':
+            case 'UsdtEth':
+            case 'UsdcStrk':
+            case 'UsdtStrk':
+            case 'UsdcSol':
+            case 'UsdtSol':
+            case 'UsdtLiquid':
+              throw new Error('Unable to send non btc assets')
+            case 'Usd':
+            case 'Eur':
+            case 'Chf':
+            case 'Mxn':
+              throw new Error('Unable to send fiat')
+          }
+        },
+        onGetPublicKey: async () => {
+          if (!svcWallet) {
+            throw new Error('Wallet not initialized')
+          }
+
+          const pk = await svcWallet.identity.compressedPublicKey()
+          return bytesToHex(pk)
+        },
+        onGetDerivationPath: () => {
+          console.log(`Called on get derivation path`)
+          // this is just a dummy one as arkade wallet uses a single key
+          return "m/84'/0'/0'/0/0"
+        },
+        onGetAddress: async (addressType: AddressType, asset?: LoanAsset) => {
+          console.log(`Called on get address: type=${addressType}, asset=${asset}`)
+
+          switch (addressType) {
+            case AddressType.ARK:
+              return arkAddress
+
+            case AddressType.BITCOIN:
+              return boardingAddress
+
+            case AddressType.LOAN_ASSET:
+              throw new Error(`Unsupported address type: ${addressType}`)
+
+            default:
+              throw new Error(`Unknown address type: ${addressType}`)
+          }
+        },
+        onGetNpub: () => {
+          console.log(`Called on get npub`)
+          // Optional - returning null for now
+          throw new Error(`NPubs are not supported`)
+        },
+        onSignPsbt: async (psbt: string) => {
+          if (!svcWallet) {
+            throw Error('Wallet not initialized')
+          }
+          const psbtBytes = hexToBytes(psbt)
+          const tx = Transaction.fromPSBT(psbtBytes, { allowUnknown: true })
```

Comment:
```suggestion
          const tx = Transaction.fromPSBT(psbtBytes)
```
If transaction is imported from arkade sdk